### PR TITLE
 [14.0][MIG] product_analytic_purchase: Migration to 14.0

### PR DESCRIPTION
**Pull request by Reyes4711-S73**
Standard migration to 14.0

### CODE REVIEW

**Review by clementmbr — APPROVED** [review]
LGTM

**Comment by clementmbr** [comment]
@rousseldenis while you merged product_analytic do you think this one can be merged too? :)

**Comment by Reyes4711-S73** [comment]
@rousseldenis Can this be merged?

**Comment by rousseldenis** [comment]
> @rousseldenis Can this be merged?

I'm not in charge here and this hasn't been approved by at least two persons.

File: product_analytic_purchase/__manifest__.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
 {
     "name": "Product Analytic Purchase",
-    "version": "10.0.1.0.0",
+    "version": "14.0.1.0.0",
     "category": "Purchases",
-    "license": "AGPL-3",
+    "license": "LGPL-3",
```
Comment (rousseldenis) [inline product_analytic_purchase/__manifest__.py]: @Reyes4711 Is there a reason to change the license here ?

Comment (Reyes4711-S73) [reply product_analytic_purchase/__manifest__.py]: @rousseldenis I made an error with this, sorry. I restore this changes. Can you approve this?

**Review by rousseldenis — APPROVED** [review]

File: product_analytic_purchase/README.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Alexis de Lattre <alexis.delattre@akretion.com>
```
Comment (rousseldenis) [inline product_analytic_purchase/README.rst:43]: @Reyes4711 You can add yourself here.

Comment (Reyes4711-S73) [reply product_analytic_purchase/README.rst:43]: @rousseldenis added

**Comment by Reyes4711-S73** [comment]
This PR could be merged? @OCA/accounting-maintainers

**Review by HaraldPanten — APPROVED** [review]
Functional review.

**Pull request merged**